### PR TITLE
refactor: FRMW-131 replacing hard-coded colors with named variables

### PR DESCRIPTION
**Pull request by 100stacks**
* [x] Dev LGTM
* [ ] Dev LGTM

Modified hard-coded hex color values with LESS named variables where available.  
https://jira.rax.io/browse/FRMW-131

### CODE REVIEW

**Comment by CITguy** [comment]
Just to double check, did you make sure to check for the uppercase versions of the hardcoded values?

**Comment by CITguy** [comment]
LGTM

File: src/rxBulkSelect/docs/rxBulkSelect.html [hunk 1]
```diff
@@ -119,9 +119,9 @@
     </script>
 
     <style>
-        .pending-status { color: #db7820; }
-        .success-status { color: #24b56a; }
-        .failure-status { color: #fc0f1d; }
+        .pending-status { color: @orange; }
+        .success-status { color: @green; }
+        .failure-status { color: @red; }
```
Comment (CITguy) [inline src/rxBulkSelect/docs/rxBulkSelect.html]: Unfortunately, LESS variables do not work in HTML.  This is one case where hard-coded values must be left alone.

Comment (CITguy) [inline src/rxBulkSelect/docs/rxBulkSelect.html]: Please revert changes to this file.

Comment (CITguy) [inline src/rxBulkSelect/docs/rxBulkSelect.html]: We'll determine a strategy to fix this in the future.

Comment (100stacks) [inline src/rxBulkSelect/docs/rxBulkSelect.html]: Okay, that is good to know.  Making the change now.

**Comment by CITguy** [comment]
@100stacks, please squash this PR. I can walk you through this, if you're not familiar with the process.

**Comment by 100stacks** [comment]
Ok will do.

**Comment by 100stacks** [comment]
Modify to use upstream/origin workflow.  Had issues with rebase. New PR https://github.com/rackerlabs/encore-ui/pull/1198

**Pull request closed without merging**